system_files/shared/usr/libexec/aurora-groups.sh

```diff
@@ -16,13 +19,16 @@ append_group() {
     grep "^$group_name:" /usr/lib/group | tee -a /etc/group > /dev/null
   fi
 }
+
 # Setup Groups
 append_group plugdev
+
 wheelarray=($(getent group wheel | cut -d ":" -f 4 | tr  ',' '\n'))
 for user in $wheelarray
```

Comment:
To iterate the array you must use `${wheelarray[@]}`

Reply:
```suggestion
for user in "${wheelarray[@]}"
```